$OpenBSD: patch-setup_py,v 1.4 2019/07/25 20:10:28 kmos Exp $

Changing to support --single-version-externally-managed

This patch has been submitted upstream and will be in the next
version of picard.

https://github.com/metabrainz/picard/pull/1210

Index: setup.py
--- setup.py.orig
+++ setup.py
@@ -4,7 +4,7 @@
 import datetime
 from distutils import log
 from distutils.command.build import build
-from distutils.command.install import install as install
+from setuptools.command.install import install as install
 from distutils.dep_util import newer
 from distutils.spawn import find_executable
 import glob
